cves/CVE-2017-17081.yml

```diff
@@ -43,11 +43,11 @@ announced_instructions: |
 
 
   Please enter your date in YYYY-MM-DD format.
-announced_date:
+announced_date: 2017-11-15
```

Comment:
Same deal here with the announced and reported dates.

Reply:
See previous conversation about the dates for this.